src/main/java/com/untamedears/ItemExchange/utility/BlockUtility.java

```diff
@@ -0,0 +1,111 @@
+package com.untamedears.ItemExchange.utility;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.block.*;
+import org.bukkit.inventory.*;
+
+import com.untamedears.ItemExchange.DeprecatedMethods;
+import com.untamedears.ItemExchange.ItemExchangePlugin;
+
+public class BlockUtility {
+
+	public static final BlockFace[] CardinalFaces = new BlockFace[] {
```

Comment:
Variable names should be lower case or in the case of static final possibly all upper

Reply:
Done.